FATEL_Backend/API/Controllers/ItemController.cs

```diff
@@ -50,4 +50,22 @@ public ActionResult<List<Item>> ReadAll()
             return StatusCode(500, e.ToString());
         }
     }
+
+    [HttpDelete]
+    [Route("Delete/{id}")]
+    public ActionResult<Item> DeleteItem(int id)
```

Comment:
```suggestion
    public ActionResult<Item> Delete([FromRoute] int id)
```
The entity name should be omitted as it is redundant, and the FromRoute should also be added here